ch_pipeline/processing/daily.py

```diff
@@ -353,20 +359,17 @@ class DailyProcessing(base.ProcessingType):
             # A good looking interval from late 2019 (determined from run
             # notes, dataflags and data availability)
             {"start": "CSD2143", "end": "CSD2148"},
-            ## Runs processed for rev_00
-            # October run
-            {"start": "20181011T140000Z", "end": "20181019T220000Z"},
-            # Winter run periods - as defined by Mateus
-            # Pass A (start trimmed for timing solutions)
-            {"start": "20181223T000000Z", "end": "20181229T000000Z"},
-            # Pass B
-            {"start": "20190111T000000Z", "end": "20190207T000000Z"},
-            # Pass C (end trimmed for timing sols)
-            {"start": "20190210T000000Z", "end": "20190304T000000Z"},
-            # April run
-            {"start": "20190406T000000Z", "end": "20190418T000000Z"},
-            # July run
-            {"start": "20190713T000000Z", "end": "20190723T000000Z"},
+        ],
+        # intervals for which we process only one day every 7 days
+        "intervals_weekly": [
+            ## dataflags and calibration tables are currently only available until October 2020
+            {"start": "20181011T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181012T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181013T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181014T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181015T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181016T000000Z", "end": "20201021T220000Z"},
+            {"start": "20181017T000000Z", "end": "20201021T220000Z"},
```

Comment:
I think it would be cleaner here to allow these specs to have a `step` field, and just put these into the main intervals list with `"step": 7`, and remove the `intervals_weekly` entirely.

Also, these definitely should be defined in terms of their CSD and not their calendar date. Over the course of a year sidereal and calendar days shift by a whole year with respect to each other, so you will get off the expected schedule at some point.

Reply:
> Also, these definitely should be defined in terms of their CSD and not their calendar date. Over the course of a year sidereal and calendar days shift by a whole year with respect to each other, so you will get off the expected schedule at some point.

I can definitely do this, I would love a little bit extra clarification! 

I chose dates instead of CSDs because there is a specific date until which we have calibration tables available for. Even with this, we still prefer CSD? Would we run into an issue where calendar days with unavailable calibration run?

Reply:
Ah, I see why you did this now.

What you have is probably fine, but as there is one more sidereal day in a year than calendar days if you pick the set of seven calendar start days, there will actually be eight sidereal days in there and the logic will miss one of them (and thus 1/8 of all days).

So you have a 1/50 chance of this breaking I think and thus probably best to specify them as CSDs.

As for your problem about calibration tables, just convert your end datetime into a CSD and use that.